### ISSUE TITLE
Check portrait or landscape?

### ISSUE THREAD

**Issue by GabrielDvt** [jenssegers/agent#90]
Sorry if I'm posting in the wrong place, I'm new to github, but I felt the need of a $agent->landscape() or $agent->portrait(), is it possible to develop?
 Thanks

**Comment by iwebnaut** [comment]
yes its not possible :D like you want it, only possible like this:


        /* render css settings on tablet */
        @if(SiteSetting::getDeviceType() == 'tablet')
                @media only screen and (orientation: landscape) {
                    .container-{{ $block->id }} {
                        padding: {{$settings->paddingTabletLandscape}};
                    }
                }

                @media only screen and (orientation: portrait) {
                    .container-{{ $block->id }} {
                        padding: {{$settings->paddingTabletPortrait}};
                    }
                }
